package.json

```diff
@@ -18,7 +18,7 @@
     "api"
   ],
   "dependencies": {
-    "unirest": "*"
```

Comment:
do you think we could do something like `0.4.*`? I'd like to make sure that the dependencies don't calcify and become unchangeable